### PR TITLE
Use devel repo everywhere, with single point of change

### PR DESCRIPTION
**Pull request by utopiabound**
Define MFL copr repo in only 1 place: /Makefile
This modifies everythwere we need MFL_* altered.

Signed-off-by: Nathaniel Clark <nclark@whamcloud.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,6 +33,8 @@ chroma-config.1.gz
 chroma_unit_test_env/
 tests/framework/utils/defaults.sh
 tests/framework/chroma_support.repo
+tests/framework/services/runner.sh
+tests/framework/integration/shared_storage_configuration/full_cluster/cluster_setup
```
Comment (jgrund) [inline .gitignore]: Should these be ignored?

File: .gitignore [hunk 2]
```diff
@@ -33,12 +33,15 @@ chroma-config.1.gz
...
 id_rsa
 id_rsa.pub
 install_build_deps-stamp
 localenv
-storage_server.repo
 base.repo
+chroma_support.repo
```
Comment (jgrund) [inline .gitignore]: Should this be ignored?

Comment (utopiabound) [reply .gitignore]: we generate it like base.repo

**Comment by johnsonw** [comment]
I noticed coverage dropped by 23.12%. Any idea why it dropped so much?

**Comment by jgrund** [comment]
The service tests are looking for:

```
$ docker exec -i systemd bash -c "./integrated-manager-for-lustre/tests/framework/services/runner.sh"
bash: ./integrated-manager-for-lustre/tests/framework/services/runner.sh: No such file or directory
```

Which appears to have been renamed to a `.in` in this patch.

File: .copr/Makefile [hunk 3]
```diff
@@ -1,15 +1,38 @@
...
+
+base.repo: base.repo.in Makefile
+
+chroma_support.repo: tests/framework/chroma_support.repo.in Makefile
+
+tests/framework/chroma_support.repo: tests/framework/chroma_support.repo.in Makefile
+
+tests/framework/utils/defaults.sh: tests/framework/utils/defaults.sh.in Makefile
+
+tests/framework/services/runner.sh: tests/framework/services/runner.sh.in Makefile
```
Comment (jgrund) [inline .copr/Makefile]: Does this need to be invoked in travis?

**Pull request merged**